modules/print-styles.js

```diff
@@ -0,0 +1,70 @@
+/* @flow */
```

Comment:
Oops, that should definitely be in there. Feel free to add it, or we'll get to it eventually :)